index.html

```diff
@@ -1777,7 +1777,10 @@ <h2 id="Collection">Backbone.Collection</h2>
     <p id="Collection-remove">
       <b class="header">remove</b><code>collection.remove(models, [options])</code>
       <br />
-      Remove a model (or an array of models) from the collection, and returns them.
+      Remove a model (or an array of models) from the collection, and returns
+      them. <tt>models</tt> (or each of its elements if an array) can be any
```

Comment:
> Personally I feel it's a little more confusing to refer to `get`. I have a slight pref to enumerate the options here.

It would be a link, the most basic building block of the web. People that are confused by that are going to have bigger problems using Backbone. Namely they're going to be confused about browsing the web let alone building it. If it's duplicated here that means that if acceptable values for `get()`'s `id` change someone will need to remember they're duplicated here and actually update them. Are you sure you don't want to keep this simple and just link to the `get()` documentation?

> Conversely, I'm not sure you need to spell out the "if models is an array" part twice. The plural kind of implies that.

It's better to be specific. For example to avoid any surprise here:

```js
// Returns model
collection.remove(model);

// Returns array
collection.remove([model]);
```